spec/models/list_import/my_anime_list/row_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+RSpec.describe ListImport::MyAnimeList::Row do
+  context 'with anime' do
+    let(:anime) { fixture('list_import/my_anime_list/toy-anime.json') }
+
+    describe '#started_at' do
+      context 'with empty date' do
+        it 'should return nil' do
+          subject = described_class.new(JSON.parse(anime)[1])
```

Comment:
nope, the issue here is that `described_class.new` is creating a new `ListImport::MyAnimeList::Row`, which expects one JSON object, rather than an array them. Basically that means the only way it will works is if I initialize it with an element in the array.